dockerfiles/html/Dockerfile

```diff
@@ -18,7 +15,16 @@ USER node
 # see https://github.com/nodejs/docker-node/blob/master/docs/BestPractices.md#global-npm-dependencies
 ENV NPM_CONFIG_PREFIX=/home/node/.npm-global
 ENV PATH=$PATH:/home/node/.npm-global/bin
-RUN npm i -g vscode-html-languageserver-bin@1.3.0
+# Ignore the warning about using WORKDIR instead of cd for convenience.
+# hadolint ignore=DL3003
+RUN cd "$HOME" \
+  && git clone https://github.com/chrismwendt/vscode-html-languageserver-bin --recursive \
```

Comment:
Can you shallow-clone only the commit?

Reply:
Yes, but that'll probably only save a few megs and won't be much faster.